coder-proxy.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
```

Comment:
I don't think we should be calling this `coder proxy` when the name is workspace proxy. It might confuse people!

I'm game for calling this `coder-workspace-proxy.service` instead. Can we toss this in `scripts` too? Top-level is a bit too exposed IMO since this feature isn't going to be extremely common.

Reply:
I'm game to move this to `scripts` if we also move `coder.service` to `scripts`.